tests/test_algorithm.py

```diff
@@ -1390,11 +1390,12 @@ def handle_data(context, data):
 
         # Starting portfolio value is 10000. Order for the asset fills on the
         # second bar of 1/06, where the price is 391, and costs the default
-        # commission of 1. On 1/07, the price is 780, and the increase in
-        # portfolio value is 780-392-1
+        # commission of 0. On 1/07, the price is 780, and the increase in
+        # portfolio value is 780-392-0
         self.assertEqual(results.port_value.iloc[0], 10000)
         self.assertAlmostEqual(results.port_value.iloc[1],
-                               10000 + 780 - 392 - 1)
+                               10000 + 780 - 392 - 0,
+                               places=2)
```

Comment:
I was getting this failure `'10387.99899997449 != 10388 within 7 places'` without adding `places=2`  I'm not sure if there is a better way to do this.

Reply:
I think this is fine.